bamboo/deploy-to-npm.sh

```diff
@@ -17,3 +17,4 @@ npx lerna publish \
   --dist-tag=${NPM_TAG} \
   --exact
 
+. ./bamboo/create-release.sh
```

Comment:
Decided to explicitly call this script post deploy-to-npm as: 

Updating the release process with another phase is workable, however for previous branches to pass we'd have to || true pass it (or detect existence and pass). 

I think the correct change is to rename both scripts to something like `publish_release_artifacts`, however I'd like to attempt that when it's less likely to impact objective work.